tools/ci/push_docs_to_repo.py

```diff
@@ -31,6 +33,11 @@
     print('Content directory does not exist')
     sys.exit(1)
 
+count = len([name for name in os.listdir(args.dir) if os.path.isfile(os.path.join(args.dir, name))])
```

Comment:
Hmm, I think I like this better than splitting it over two lines, even though linting objects